scico/solver.py

```diff
@@ -727,7 +727,7 @@ def __init__(
         else:
             D = jnp.array(D)
             if not D.ndim in [1, 2]:
-                raise ValueError("If matrix, D should be 1D or 2D.")
+                raise ValueError("If MatrixOperator, D should be 1D or 2D.")
```

Comment:
This error can be raised by either an array or a MatrixOperator, maybe should be "If array or MatrixOperator..."

Reply:
Fixed.